spec/lib/efile/id/id40_calculator_spec.rb

```diff
@@ -58,6 +57,7 @@
 
   describe "Line 6d: Total Exemptions" do
     it "sums lines 6a, 6b, and 6c" do
+      allow(instance).to receive(:line_or_zero).and_return(nil)
```

Comment:
the purpose of this?

Reply:
This is necessary since we now call `line_or_zero` in the calculator with params other than the ones specified here, and if you don't provide a default you get an error saying that one is required:
```
RSpec::Mocks::MockExpectationError: #<Efile::Id::Id40Calculator ...> received :line_or_zero with unexpected arguments
  expected: (:ID40_LINE_6C)
  got: (:ID40_LINE_43_WORKSHEET)
...
Please stub a default value first if message might be received with other args as well. 
```

If you know of another way to deal with this, let me know! But afaik, once you mock any return values for a method, no other calls to that method will ever get to the actual code and also must return a mocked value.

Reply:
Nevermind, found a way to have it call the original code by default!